#goal
nav_msgs/Path path
---
#result
bool success
---
#feedback
float64 percent_finished
